src/expressions/dataTypes/Value.ts

```diff
@@ -1,5 +1,8 @@
+// The actual type is type ValueValue = NodePointer | Function | string | number | boolean | QName | Duration | DateTime; but doing that gives us thousands of errors.
+type ValueValue = any;
```

Comment:
Than why do we add a type for it now? We could as well just leave it as any and add a todo / issue to fix this at some point.

Reply:
Meh, I'm going to keep it